IGC/AdaptorOCL/CMakeLists.txt

```diff
@@ -6,9 +6,17 @@ set(IGC_BUILD__PROJ_LABEL__AdaptorOCL "AdaptorOCL")
 include_directories("${CMAKE_CURRENT_SOURCE_DIR}")
 #include_directories("${CMAKE_CURRENT_SOURCE_DIR}/ocl_igc_shared/executable_format")
 
+set(ver LLVM_VERSION_MAJOR)
```

Comment:
This has to be ${LLVM_VERSION_MAJOR}

Reply:
yep. thank you for catching this. I eventually tried with llvm-9 and forgot other cases.

Reply:
@chivakker : fixed. PR updated.